Cue/app/common/DeckThumbnail.js

```diff
@@ -57,6 +86,22 @@ export default class DeckThumbnail extends React.Component {
       // Display "Updating..."?
     }
 
+    let inset
+    if (!this.props.hideInsets) {
+      if (this.props.deck.accession === 'private') {
+        if (this.props.deck.public) {

```

Comment:
Hmm, I don't think so, because it doesn't add any detail. Public decks in the user's library are already sorted under the "Public" heading so it would just be redundant information.